libs/ngx-charts-on-fhir/src/index.ts

```diff
@@ -19,6 +19,7 @@ export * from './lib/dynamic-table/dynamic-table.component';
 export * from './lib/fhir-data/fhir-data.service';
 export * from './lib/data-layer-browser/data-layer-browser.component';
 export * from './lib/data-layer-list/data-layer-list/data-layer-list.component';
+export * from './lib/data-layer-list/shared-data-layer-list.service';
```

Comment:
Why does this need to added to the public API? It's only used for internal communication between components, right?